CHANGELOG.next.asciidoc

```diff
@@ -61,7 +61,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...main[Check the HEAD dif
 - Update documentation for accessing `last_response.url.params` in httpjson input. {pull}30739[30739]
 
 *Filebeat*
-
+- Fixes data duplication on restart when filestream inputs did not have an ID set. {issue}30061[30061]
```

Comment:
We will have to reword this (and remove the breaking change note above). I would be as detailed as possible about what the issue is and how to fix it.

Reply:
What do you think of the new version? Technically it's not a full bug fix because the users can still make it happen by not setting IDs despite all the warnings in the logs.